connector-runtime/connector-runtime-application/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM eclipse-temurin:17.0.7_7-jre
+
+RUN mkdir /opt/app
+
+COPY start.sh /start.sh
+RUN chmod +x start.sh
+
+# Copy Connector runtime from local build
+COPY target/*-with-dependencies.jar /opt/app/
```

Comment:
Maybe it even makes sense to simplify it to `*.jar`. WDYT?

Reply:
This way the non-executable slim jar will also make it to the docker image.I think it makes sense to keep the classpath clean and only add the needed files